src/lightning/data/streaming/cache.py

```diff
@@ -40,6 +49,8 @@ def __init__(
             cache_dir: The path to where the chunks will be stored.
             remote_dir: The path to a remote folder where the data are located.
                 The scheme needs to be added to the path.
+            name: The name of dataset in the cloud.
```

Comment:
can you add what the default corresponds to in the docstrings? Like if I don't pass `name=...` what will happen?

Reply:
Will do in another PR.